src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/ByteArrayToStorageQueueMessageConverter.cs

```diff
@@ -1,34 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using Microsoft.Azure.WebJobs.Host.Converters;
-using Microsoft.Azure.WebJobs.Host.Storage.Queue;
-
-namespace Microsoft.Azure.WebJobs.Host.Queues.Bindings
-{
-    internal class ByteArrayToStorageQueueMessageConverter : IConverter<byte[], IStorageQueueMessage>
-    {
-        private readonly IStorageQueue _queue;
-
-        public ByteArrayToStorageQueueMessageConverter(IStorageQueue queue)
-        {
-            if (queue == null)
-            {
-                throw new ArgumentNullException("queue");
-            }
-
-            _queue = queue;
-        }
-
-        public IStorageQueueMessage Convert(byte[] input)
-        {
-            if (input == null)
-            {
-                throw new InvalidOperationException("A queue message cannot contain a null byte array instance.");
```

Comment:
What do these custom conversion error strings get replaced in the new converter pipeline with out of curiosity?